docs/settings/security-settings.asciidoc

```diff
@@ -342,131 +252,107 @@ xpack.security.audit.appender: <1>
   layout:
     type: json
 ----------------------------------------
+
 <1> This appender is the default and will be used if no `appender.*` config options are specified.
 <2> Rotates log files every 24 hours.
 <3> Keeps maximum of 10 log files before deleting older ones.
 
-| `xpack.security.audit.appender`
-| Optional. Specifies where audit logs should be written to and how they should be formatted. If no appender is specified, a default appender will be used (see above).
+xpack.security.audit.appender::
+Optional. Specifies where audit logs should be written to and how they should be formatted. If no appender is specified, a default appender will be used (see above).
 
-| `xpack.security.audit.appender.type`
-| Required. Specifies where audit logs should be written to. Allowed values are `console`, `file`, or `rolling-file`. 
+xpack.security.audit.appender.type::
+Required. Specifies where audit logs should be written to. Allowed values are `console`, `file`, or `rolling-file`. 
 
 Refer to <<audit-logging-file-appender>> and <<audit-logging-rolling-file-appender>> for appender specific settings.
 
-| `xpack.security.audit.appender.layout.type`
-| Required. Specifies how audit logs should be formatted. Allowed values are `json` or `pattern`.
+xpack.security.audit.appender.layout.type::
+Required. Specifies how audit logs should be formatted. Allowed values are `json` or `pattern`.
```

Comment:
In addition to the indent issues that Gail pointed out:

![image](https://user-images.githubusercontent.com/5295965/151807766-3f3940a2-29c4-4377-ae4f-b08c660ab6a9.png)

* L260-261 should be indented less
* L263-264 should be indented less
* L266 should be indented more (under the setting)